src/composables/canvas/useSelectionToolboxPosition.ts

```diff
@@ -168,50 +168,58 @@ export function useSelectionToolboxPosition(
     }
   )
 
-  // Watch for dragging state
-  watch(
-    () => canvasStore.canvas?.state?.draggingItems,
-    (dragging) => {
-      if (dragging) {
-        visible.value = false
-
-        if (moreOptionsOpen.value) {
-          const currentSig = buildSelectionSignature(canvasStore)
-          if (currentSig !== moreOptionsSelectionSignature) {
-            moreOptionsSelectionSignature = null
-          }
-          moreOptionsWasOpenBeforeDrag = true
-          moreOptionsOpen.value = false
-          moreOptionsRestorePending.value = !!moreOptionsSelectionSignature
-          if (moreOptionsRestorePending.value) {
-            forceCloseMoreOptionsSignal.value++
-          } else {
-            moreOptionsWasOpenBeforeDrag = false
-          }
+  // Helper function to handle drag state changes (shared between LiteGraph and Vue nodes)
+  const handleDragStateChange = (dragging: boolean) => {
+    if (dragging) {
+      visible.value = false
+
+      if (moreOptionsOpen.value) {
+        const currentSig = buildSelectionSignature(canvasStore)
+        if (currentSig !== moreOptionsSelectionSignature) {
+          moreOptionsSelectionSignature = null
+        }
+        moreOptionsWasOpenBeforeDrag = true
+        moreOptionsOpen.value = false
+        moreOptionsRestorePending.value = !!moreOptionsSelectionSignature
+        if (moreOptionsRestorePending.value) {
+          forceCloseMoreOptionsSignal.value++
         } else {
-          moreOptionsRestorePending.value = false
           moreOptionsWasOpenBeforeDrag = false
         }
       } else {
-        requestAnimationFrame(() => {
-          updateSelectionBounds()
-          const selectionMatches = currentSelectionMatchesSignature(canvasStore)
-          const shouldRestore =
-            moreOptionsWasOpenBeforeDrag &&
-            visible.value &&
-            moreOptionsRestorePending.value &&
-            selectionMatches
-
-          if (shouldRestore) {
-            restoreMoreOptionsSignal.value++
-          } else {
-            moreOptionsRestorePending.value = false
-          }
-          moreOptionsWasOpenBeforeDrag = false
-        })
+        moreOptionsRestorePending.value = false
+        moreOptionsWasOpenBeforeDrag = false
       }
+    } else {
+      requestAnimationFrame(() => {
+        updateSelectionBounds()
+        const selectionMatches = currentSelectionMatchesSignature(canvasStore)
+        const shouldRestore =
+          moreOptionsWasOpenBeforeDrag &&
+          visible.value &&
+          moreOptionsRestorePending.value &&
+          selectionMatches
+
+        if (shouldRestore) {
+          restoreMoreOptionsSignal.value++
+        } else {
+          moreOptionsRestorePending.value = false
+        }
+        moreOptionsWasOpenBeforeDrag = false
+      })
     }
-  )
+  }
+
+  // Unified dragging state - combines both LiteGraph and Vue node dragging
+  const isDragging = computed(() => {
```

Comment:
To protect against the missing `.value` thing
```suggestion
  const isDragging = computed((): boolean => {
```